### PR TITLE
Story #13901 feat(agencies): add create and update button in creation form

### PR DESCRIPTION
**Pull request by Regzox**
## Description

* Ajout du bouton de création et redirection vers l'édition.

### CODE REVIEW

File: ui/ui-frontend/projects/referential/src/app/agency/agency-create/agency-create.component.ts [hunk 1]
```diff
@@ -92,16 +92,21 @@ export class AgencyCreateComponent implements OnInit {
...
     this.isLoading = true;
-    this.agencyService.create(this.form.value).subscribe(
-      () => {
-        this.dialogRef.close({ success: true, action: 'restart' });
+    this.agencyService.create(agency as Agency).subscribe({
+      next: () => {
+        this.dialogRef.close({ success: true, action: 'edit', agency });
       },
-      (error: any) => {
-        this.isLoading = false;
```
Comment (marob) [inline ui/ui-frontend/projects/referential/src/app/agency/agency-create/agency-create.component.ts:102]: Pourquoi ne pas arrêter le loader en cas d'erreur ?

Comment (Regzox) [reply ui/ui-frontend/projects/referential/src/app/agency/agency-create/agency-create.component.ts:102]: Un oubli, je voulais passer par du tap, finalize... Mais comme le composant est instancié à chaque ajout le problème de gestion d'état que j'ai voulu corriger ne se voit pas.

**Comment by vitam-prg** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [be12990a\-21c2\-4b11\-b15c\-1b92d99fcc53](https://eu.ast.checkmarx.net/projects/3f06ff53-d733-40ac-a6fb-49e415cb793f/scans?id=be12990a-21c2-4b11-b15c-1b92d99fcc53&branch=story_13901)
### New Issues
Severity|Issue|Source File / Package|Checkmarx Insight
---|---|---|---
LOW|Unpinned Actions Full Length Commit SHA|/build\-and\-test\.yml: [99](https://github.com/ProgrammeVitam/vitam-ui/blob/story_13901//.github/workflows/build-and-test.yml#L99)|Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release. Pinning to a particular SHA helps...
LOW|Unpinned Actions Full Length Commit SHA|/build\-and\-test\.yml: [153](https://github.com/ProgrammeVitam/vitam-ui/blob/story_13901//.github/workflows/build-and-test.yml#L153)|Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release. Pinning to a particular SHA helps...
LOW|Unpinned Actions Full Length Commit SHA|/build\-and\-test\.yml: [158](https://github.com/ProgrammeVitam/vitam-ui/blob/story_13901//.github/workflows/build-and-test.yml#L158)|Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release. Pinning to a particular SHA helps...
### Fixed Issues
Severity|Issue|Source File / Package
---|---|---
LOW|Unpinned Actions Full Length Commit SHA|/build\-and\-test\.yml: [100](https://github.com/ProgrammeVitam/vitam-ui/blob/story_13901//.github/workflows/build-and-test.yml#L100)
LOW|Unpinned Actions Full Length Commit SHA|/build\-and\-test\.yml: [154](https://github.com/ProgrammeVitam/vitam-ui/blob/story_13901//.github/workflows/build-and-test.yml#L154)
LOW|Unpinned Actions Full Length Commit SHA|/build\-and\-test\.yml: [159](https://github.com/ProgrammeVitam/vitam-ui/blob/story_13901//.github/workflows/build-and-test.yml#L159)

**Review by marob — APPROVED** [review]

File: ui/ui-frontend/projects/referential/src/app/agency/agency-preview/agency-preview.component.html [hunk 2]
```diff
@@ -4,6 +4,19 @@
   [icon]="'vitamui-icon-agent'"
   (onclose)="emitClose()"
 >
+  <vitamui-common-menu-button [overlayPos]="'end'" [icon]="'vitamui-icon-more-horiz'">
+    <button
+      mat-menu-item
+      [matTooltip]="'AGENCY.ACTION.EDIT' | translate"
+      matTooltipClass="vitamui-tooltip"
+      [matTooltipShowDelay]="300"
```
Comment (marob) [inline ui/ui-frontend/projects/referential/src/app/agency/agency-preview/agency-preview.component.html]: Utiliser vitamuiCommonToolTip

**Review by marob — CHANGES_REQUESTED** [review]

**Review by laedanrex — APPROVED** [review]

**Review by marob — APPROVED** [review]

**Pull request merged**